external-crates/move/crates/move-core-types/src/resolver.rs

```diff
@@ -26,11 +26,17 @@ pub struct TypeOrigin {
 /// storage IDs.
```

Comment:
Should this come live in `move-vm-runtime` eventually?

Reply:
Possilby, or other stuff from the `move-vm-runtime` will need to move here (or some in-between state). We're going to need to have an execution versioning discussion about the `move-vm-runtime` crate and where types live as that was cropping up higher up as well for some other types.